.travis.yml

```diff
@@ -17,6 +17,8 @@ script:
   - bazel test --noshow_progress --noshow_loading_progress --test_output=errors //src/wasm/... //src/tests/...
   - bazel build --noshow_progress --noshow_loading_progress //particles/Native/Wasm:all
   - bazel build --noshow_progress --noshow_loading_progress //particles/Tutorial/Kotlin:all
+  - cd src_kt
+  - bazel test --noshow_progress --noshow_loading_progress --test_output=errors --incompatible_depset_is_not_iterable=false //javatests/...
```

Comment:
I think we will need to unify the .bazelproject files in this case, otherwise Android Studio barfs when trying to import more than one .bazelproject from the same directory (the top-level arcs directory), since there's already a `.aswb` directory after the first project import.  

I had tried merging the project files into one, but Android Studio was complaining about the package identifiers in the kotlin and java files when that happened - saying that packages shoudl be `src_kt.java.arcs.whatever`, instead of `arcs.whatever`.   I'm starting to think we should probably have all JVM code in the same root directories to avoid this situation.